earth_enterprise/src/version.txt

```diff
@@ -1 +1,5 @@
+
 5.2.4
+
+5.2.4
```

Comment:
Can you remove the changes to version.txt? That must have been a mistake

Reply:
This is actually pretty weird. It doesn't say that any of your commits added these lines. I think you may just need to merge master into the feature branch to clear this up.